test/bad_params.fromasm

```diff
@@ -3,7 +3,7 @@
  (data (global.get $__memory_base) "bad_params.asm.js")
  (import "env" "__memory_base" (global $__memory_base i32))
  (export "ex" (func $ex))
- (func $ex (; 0 ;) (; has Stack IR ;)
+ (func $ex (; 0 ;)
```

Comment:
This is relevant because this pass now invalidates the stack IR because it is able to modify the IR. @kripken did we need to do something about this?

Reply:
In general if stack IR is no longer present we need to understand why. I'm not sure why `bad_params` would have stack IR, though, so that's the first question. This is from asm.js, so I suspect perhaps this PR has made asm2wasm no longer emit stack IR, which would be bad?

If that's it then maybe this pass needs to run at a different time, so it doesn't invalidate stack IR.

Reply:
This should be fixed now.